config/src/config/state_sync_config.rs

```diff
@@ -122,9 +122,9 @@ pub struct StateSyncDriverConfig {
 impl Default for StateSyncDriverConfig {
     fn default() -> Self {
         Self {
-            bootstrapping_mode: BootstrappingMode::ApplyTransactionOutputsFromGenesis,
+            bootstrapping_mode: BootstrappingMode::ExecuteOrApplyFromGenesis,
             commit_notification_timeout_ms: 5000,
-            continuous_syncing_mode: ContinuousSyncingMode::ApplyTransactionOutputs,
+            continuous_syncing_mode: ContinuousSyncingMode::ExecuteTransactionsOrApplyOutputs,
```

Comment:
should Intelligent or AutoSelect be in the name? :)

Reply:
Aah, I'm still so-so on the term `intelligent` 😄 Also, these configs are currently pretty technical 😄, e.g., `fast sync` is actually called `DownloadLatestStates` in the config. When we write the docs, we can use the informal names (e.g., https://aptos.dev/guides/state-sync#fast-syncing), but we can always follow up once we think the informal names make sense (long term).